### DISCUSSION TITLE
Does hurlfile support `skip` semantics?

### DISCUSSION THREAD

**Discussion by chenrui333** [Orange-OpenSource/hurl#1810] [Q&A]
I am curious if I can `skip` the test cases in the hurlfile or skip the entire file.

**Comment by jcamiel** [comment]
Hi @chenrui333 not currently but it's surely a good idea. We think about adding a kind of header also to specify option for the whole file (no syntax idea for the moment) and `skip` could fit into.

**Reply by chenrui333** [reply]
Cool!

**Reply by chenrui333** [reply]
maybe we can convert this discussion into a feature request :)

**Reply by chenrui333** [reply]
Thanks @jcamiel!
